theano/gof/cmodule.py

```diff
@@ -1679,15 +1681,19 @@ def std_lib_dirs_and_libs():
         # get the name of the python library (shared object)
         libname = distutils.sysconfig.get_config_var("LDLIBRARY")
 
-        if libname.startswith("lib"):
-            libname = libname[3:]
+        if libname:
+            if libname.startswith("lib"):
+                libname = libname[3:]
 
-        # remove extension if present
-        if libname.endswith(".so"):
-            libname = libname[:-3]
-        elif libname.endswith(".a"):
-            libname = libname[:-2]
+            # remove extension if present
+            if libname.endswith(".so"):
+                libname = libname[:-3]
+            elif libname.endswith(".a"):
+                libname = libname[:-2]
+        else:
```

Comment:
Here, to be safer, I would go with something like:
elif ((this is a pypy interpretor, you described how to do it.):
   libname = "pypy-c"